sites/friday/src/add-services/_index.md

```diff
@@ -162,10 +162,13 @@ title=In an app
 +++
 
 Once a service is running and exposed as a relationship,
-its credentials (such as the host, username, and password) are available through the `{{% vendor/prefix %}}_RELATIONSHIPS` environment variable.
+its credentials (such as the host, username, and password) are automatically available as [service environment variables](/development/variables.html#service-specific-variables), with the pattern:
+```
+{{< variable "<CAPITALIZED_RELATIONSHIP_NAME>" >}}_<VARIABLE>=<VALUE>
+```
```

Comment:
We have a display issue in the output here, so I think it's better if we use inline variables rather than a code block. And I used the ${{< variable "RELATIONSHIP_NAME" >}}-_{{< variable "SERVICE_PROPERTY" >}} format because that's how Jérôme described it in his design doc and I think that might be more straightforward, WDYT?

```suggestion
its credentials (such as the host, username, and password) are automatically available as [service environment variables](/development/variables.html#service-specific-variables), in the ${{< variable "RELATIONSHIP_NAME" >}}-_{{< variable "SERVICE_PROPERTY" >}} format.
```